torch_geometric/loader/neighbor_loader.py

```diff
@@ -26,6 +26,7 @@ def __init__(
         input_type: Optional[Any] = None,
         share_memory: bool = False,
         time_attr: Optional[str] = None,
+        keep_data=False,
```

Comment:
I personally prefer `is_sorted` argument, which dictates that `edge_index` is sorted by destination. Then we simply need to run `torch.ops.torch_sparse.ind2ptr` to compute the `colptr`.
 ```suggestion
        is_sorted: bool=False,
```